dockerfiles/jupyter/Dockerfile

```diff
@@ -25,6 +25,15 @@ RUN apt-get update && \
 RUN apt-get update && apt-get install --no-install-recommends -y \
   graphviz \
  && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && apt-get install -y curl apt-transport-https
+# Download and install kubectl
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
```

Comment:
why is this needed?

Reply:
this is kubectl in order to be able to  interact with k8s cluster